server/dataParser/reviews.js

```diff
@@ -0,0 +1,57 @@
+const axios = require('axios');
+const Promise = require('bluebird');
+let url = 'https://app-hrsei-api.herokuapp.com/api/fec2/rfp';
+
+var config = {
+  headers: {
+    Authorization: process.env.GIT_TOKEN
+  }
+}
+
+module.exports = {
+  fetchReviewsData (productID = '65632', page=1, count=5, sort) {
+    return new Promise ((resolve, reject) => {
+      if (sort) {
+        axios.get(`${url}/reviews?product_id=${productID}&page=${page}&count=${page}&sort=${sort}`, config)
```

Comment:
It might be slightly more readable to use params (https://masteringjs.io/tutorials/axios/get-query-params) instead of making this long URL. Ultimately, they do the same exact thing, but when the URL gets kind of long and there are lots of parameters, readability could become an issue.